commit 8de6e4a199ba6cc8aaeb43924b974eed67164bd6
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Sat Feb 5 11:06:01 2022 -0800

    x86: Improve L to support L(XXX_SYMBOL (YYY, ZZZ))
    
    (cherry picked from commit 1283948f236f209b7d3f44b69a42b96806fa6da0)

diff --git a/sysdeps/x86/sysdep.h b/sysdeps/x86/sysdep.h
index 937180c1bd791570..deda1c4e492f6176 100644
--- a/sysdeps/x86/sysdep.h
+++ b/sysdeps/x86/sysdep.h
@@ -111,7 +111,8 @@ enum cf_protection_level
 /* Local label name for asm code. */
 #ifndef L
 /* ELF-like local names start with `.L'.  */
-# define L(name)	.L##name
+# define LOCAL_LABEL(name) .L##name
+# define L(name)	LOCAL_LABEL(name)
 #endif
 
 #define atom_text_section .section ".text.atom", "ax"
